lib/restrictions.js

```diff
@@ -196,6 +223,22 @@ export async function remove({zone, resource} = {}) {
     'restriction.resource': resource
   };
   const collection = database.collections['resource-restriction-restriction'];
+  await collection.deleteMany(query);
+}
+
+/**
+ * Deletes a restriction from the database by ID.
+ *
+ * @param {object} options - The options to use.
+ * @param {string} options.id - The ID of restriction.
+ *
+ * @returns {Promise} - Settles once the operation completes.
+ */
+export async function removeById({id}) {
```

Comment:
I think our convention is to just name this `remove`.

Reply:
Updated to follow the same convention, it is another breaking change as `remove` now requires an `id` and `removeAll` now takes the `zone` and `resource` (which is now the same pattern as `get`).